cypress/fixtures/analysis.json

```diff
@@ -1,51 +1,58 @@
-[
-    {
+{
+    "Book-server": {
         "source": "Source code",
         "target": ["Containerization"],
         "appName": "book-server",
         "storyPoints": 5
     },
-    {
+
+    "Tackle-testapp-2-targets": {
         "source": "Source code + dependencies",
         "target": ["Jakarta EE 9", "Containerization"],
         "appName": "tackle-testapp",
         "storyPoints": 1
     },
-    {
+
+    "Daytrader-app-SourceCode-dependencies": {
         "source": "Source code + dependencies",
         "target": ["Jakarta EE 9", "Containerization"],
         "appName": "deps",
         "storyPoints": 24
     },
-    {
+
+    "Customers-tomcat": {
         "source": "Binary",
         "target": ["Containerization"],
         "appName": "customers-tomcat-0.0.1-SNAPSHOT.war",
         "storyPoints": 2
     },
-    {
+
+    "JakartaEE9": {
         "source": "Upload a local binary",
         "target": ["Jakarta EE 9"],
         "binary": ["acmeair-webapp-1.0-SNAPSHOT.war"],
         "appName": "acmeair-webapp-1.0-SNAPSHOT.war",
         "storyPoints": 78
     },
-    {
+
+    "Customers-tomcat-upload-binary": {
         "source": "Upload a local binary",
         "target": ["Jakarta EE 9"],
         "binary": ["customers-tomcat-0.0.1-SNAPSHOT.war"],
         "customRule": "javax-package-custom-target.windup.xml",
         "appName": "customers-tomcat-0.0.1-SNAPSHOT.war",
         "storyPoints": 21
     },
-    {
+
+    "Tackle-testapp": {
```

Comment:
we can remove this data and use same tackle-testapp with 2 targets and add enableTagging flag there .

Reply:
For ```Tackle-testapp-2-targets": {
        "source": "Source code + dependencies", ``` . tackle-testapp here is used with ``` "source": "Source code",```